web/templates/includes/panel.html

```diff
@@ -102,25 +100,33 @@
             <li class="top-bar-nav-item">
               <a title="<?=_('Logs');?>" class="top-bar-nav-link <?php if($TAB == 'LOG') echo 'active' ?>" href="/list/log/">
                 <i class="fas fa-clock-rotate-left"></i>
-                <span class="u-hidden"><?=_('Logs');?></span>
+                <span class="top-bar-nav-link-label"><?=_('Logs');?></span>
               </a>
             </li>
           <?php } else { ?>
             <?php if ($panel[$user]['SUSPENDED'] === 'no') {?>
               <li class="top-bar-nav-item">
                 <a title="<?=htmlspecialchars($user)?> (<?=htmlspecialchars($panel[$user]['NAME'])?>)" class="top-bar-nav-link" href="/edit/user/?user=<?=$user; ?>&token=<?=$_SESSION['token']?>">
```

Comment:
Active is not "active" when you click on "User (xxxx") and the page is reloaded.

Reply:
Yeah, existing issue on 1.6 🤷

Reply:
It doesn't surprise me.